docs/contributing/translator_onboarding.md

```diff
@@ -0,0 +1,35 @@
+# ONBOADING GUIDE FOR TRANSLATORS
+
+   Wagtail uses transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
+## Translation workflow overview
+   Each new release has about two weeks release candidate period. At the start of that period the translations are put into transifex, we notify the channel in slack that new translations are needed. Those are then updated when there is a release candidate until the final release. If translations reach a threshold of X% ( this is about 90) we communicate that this is an officially supported translation in the docs.

```

Comment:
```suggestion
Translations from Transifex are only integrated into the Wagtail repository at the time of a new release. When a release is close to being ready there will be a RC (Release Candidate) for the upcoming version and the translations will be exported to Transifex.

During this RC period, usually around two weeks, there will be a chance for all the translators to update and add new translations. We will also notify the `#translators` channel in the Watail slack group at this time.

These new translations are imported into Wagtail for any subsequent RC and the final release.

If translations reach a threshold of about 90% we communicate that this is an officially supported translation in the docs.
```

Have a read, and see what you think.

Reply:
Sorry. It's actually 90% as first suggested. 

As per https://github.com/wagtail/wagtail/wiki/Managing-Wagtail-translations#fetching-new-translations-from-transifex